src/mcp_server_helm/core/server.py

```
@@ -0,0 +1,281 @@
+        """
+        Main function to run the MCP server for Helm commands.
+        """
+        logger.info("Starting Helm MCP server")
+
+        logger.info("Creating initialization options")
+        options = self.server.create_initialization_options()
+
+        logger.info("Starting stdio server")
+        async with stdio_server() as (read_stream, write_stream):
+            logger.info("Running MCP server")
+            await self.server.run(read_stream, write_stream, options, raise_exceptions=True)
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)